ui/src/app/pages/analysis/util.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import { mount } from 'enzyme';
+
+import { WorkspaceAccessLevel } from 'generated/fetch';
+
+import { NotebookResourceCard } from 'app/pages/analysis/notebook-resource-card';
+
+import { appendNotebookFileSuffixByOldName } from './util';
+
+describe('NotebookUtil', () => {
```

Comment:
please add a case for what happens when it's neither of the correct ones

Reply:
The current code is `if jupyter append jpynb, else append {rmd}`. I can add `else if rmd, append rmd` but I am not sure how to handle the else case when neither of the correct ones. 
Does don't append anything SG?

Reply:
yes - that should match the recent change to the function